### PR TITLE
feat: add important meta elements to improve seo 🚀

### PR DESCRIPTION
**Pull request by 0xabdulkhalid**
## Related Issue

Closes: #501 

## Description of Changes
- This PR is about adding important meta elements for improving the site's seo score along to boost site's vital performance
- Meta elements in HTML play a crucial role in providing information about the webpage to browsers and search engines. 
- They are typically placed in the head section of an HTML document and are not displayed directly on the webpage.
- Those Meta elements in HTML serve various purposes, including defining document encoding, improving search engine visibility, enhancing social media sharing, optimizing for mobile devices, and influencing browser behavior. 
- Utilizing appropriate meta elements can significantly impact how webpages are perceived, indexed, and shared on the internet.

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

### CODE REVIEW

**Comment by anmode** [comment]
errors are there.. try to build locally
```


./pages/_app.page.js:68:8
--
19:46:54.050 | Syntax error: Unexpected token
19:46:54.050 |  
19:46:54.050 | 66 \|         />
19:46:54.050 | 67 \|
19:46:54.050 | > 68 \|        <!-- SEO ESSENTIALS -->
19:46:54.050 | \|         ^
19:46:54.050 | 69 \|         <meta
19:46:54.050 | 70 \|             name="robots"
19:46:54.050 | 71 \|             content="index, follow"


```

**Comment by 0xabdulkhalid** [comment]
Sorry for the inconvience, i will make those changes after evening.

Cuz, I'm way to my college.

**Comment by anmode** [comment]
> Sorry for the inconvience, i will make those changes after evening.
> 
> Cuz, I'm way to my college.

no worries, when you get time carry on :)

**Comment by 0xabdulkhalid** [comment]
> **The latest updates on your projects**. Learn more about [Vercel for Git ↗︎](https://vercel.link/github-learn-more)
> Name 	Status 	Preview 	Comments 	Updated (UTC)
> **grabtern-frontend** 	white_check_mark Ready ([Inspect](https://vercel.com/anmode/grabtern-frontend/AEtd1bw19NmvEACE2uTC1kK3aZyL)) 	[Visit Preview](https://vercel.live/open-feedback/grabtern-frontend-git-fork-0xabdulkhalid-closes-i-cd2bfd-anmode.vercel.app?via=pr-comment-visit-preview-link&passThrough=1) 	speech_balloon [**Add feedback**](https://vercel.live/open-feedback/grabtern-frontend-git-fork-0xabdulkhalid-closes-i-cd2bfd-anmode.vercel.app?via=pr-comment-feedback-link) 	Jul 16, 2023 3:26am

@anmode Now's the Build has been successfully deployed!

**Comment by 0xabdulkhalid** [comment]
Hey there @anmode, now you can check the Build.

There's no error now.

**Comment by anmode** [comment]
Sure!!

File: pages/_app.page.js [hunk 1]
```diff
@@ -56,55 +56,145 @@ function MyApp({ Component, pageProps }) {
...
+            name="googlebot" 
+            content="index, follow, max-snippet:-1, max-image-preview:large, max-video-preview:-1"
+        />
+        <meta 
+            http-equiv="content-language" 
+            content="en"
+        />
+        <meta 
+            name="description"
+            content="With GrabTern you can easly get your dream Intern from GSoc, MLH, Amazon ML Summer Intern and Many more that will be guide by mentor only just for ₹ 1 Rupee, So What are you waiting for? Sign Up the intern at GrabTern and will be guide by mentor and dream your intern job sooner!"
```
Comment (maaz-bin-asad) [inline pages/_app.page.js:97]: Spelling mistakes and reframing the content to make it more appealing.

File: pages/_app.page.js [hunk 2]
```diff
@@ -56,55 +56,145 @@ function MyApp({ Component, pageProps }) {
...
+        />
+        <meta 
+            property="twitter:title" 
+            content="Grabtern" 
         />
         <meta
-          property="instagram:username"
-          content="https://www.instagram.com/grabtern.guide/"
+            property="twitter:description"
+            content="With GrabTern you can easly get your dream Intern from GSoc, MLH, Amazon ML Summer Intern and Many more that will be guide by mentor only just for ₹ 1 Rupee, So What are you waiting for? Sign Up the intern at GrabTern and will be guide by mentor and dream your intern job sooner!"
```
Comment (maaz-bin-asad) [inline pages/_app.page.js:197]: Same here. Check the content again

File: pages/_app.page.js [hunk 3]
```diff
@@ -56,55 +56,145 @@ function MyApp({ Component, pageProps }) {
...
+            http-equiv="content-language" 
+            content="en"
+        />
+        <meta 
+            name="description"
+            content="With GrabTern you can easly get your dream Intern from GSoc, MLH, Amazon ML Summer Intern and Many more that will be guide by mentor only just for ₹ 1 Rupee, So What are you waiting for? Sign Up the intern at GrabTern and will be guide by mentor and dream your intern job sooner!"
+        />
+        <meta 
+            name="keywords"
+            content="Intern, Internships, GSoc, MLH, Amazon ML Summer Intern, Outreachy, GrabTern, Internship, Adobe SHE Codes"
```
Comment (maaz-bin-asad) [inline pages/_app.page.js:101]: Add "Mentor" related keywords. Besides, keywords should always be specific

**Review by maaz-bin-asad — CHANGES_REQUESTED** [review]

**Comment by Abhinavcode13** [comment]
@0xabdulkhalid Are you working?

**Comment by 0xabdulkhalid** [comment]
Yeah, I'm here @Abhinavcode13.

**Comment by 0xabdulkhalid** [comment]
I just busy due to attending my college, i will continue this work tonight.

**Comment by 0xabdulkhalid** [comment]
Hey there @maaz-bin-asad, i have resolved those changes you need.

Now you can verify them once again.

**Comment by 0xabdulkhalid** [comment]
> Can we have other SEO improvements in this pR only? I have tested with some SEO tools. Grabtern needs to improve SEO. Please look into it.

I can't get your thoughts about "other SEO", i think this PR is enough for SEO Improvements.

Because i have covered every important meta elements to meet the right aspects for SEO.

**Comment by 0xabdulkhalid** [comment]
Can i know why the deployment's UI is differ from the default grabtern UI ?

**Review by maaz-bin-asad — APPROVED** [review]

**Comment by maaz-bin-asad** [comment]
> Hey there @maaz-bin-asad, i have resolved those changes you need.
> 
> Now you can verify them once again.

Approved

**Comment by anmode** [comment]
> > Can we have other SEO improvements in this pR only? I have tested with some SEO tools. Grabtern needs to improve SEO. Please look into it.
> 
> I can't get your thoughts about "other SEO", i think this PR is enough for SEO Improvements.
> 
> Because i have covered every important meta elements to meet the right aspects for SEO.
> 
> **EDIT**: Now i can get your words @anmode, the SEO is affected due to non-crawlable links and images without proper `alt` attributes. We can make another issue and related PR for that. For now this PR is for adding Meta.
 

yess, ok you can cover them in other PR.
and actually deployment UI will be new UI for grabtern.. I'll soon put it into production after some bug fixes and improvment.
so think like deployment version is new release for grabtern.

**Comment by anmode** [comment]
> > Hey there @maaz-bin-asad, i have resolved those changes you need.
> > Now you can verify them once again.
> 
> Approved

Thanks @maaz-bin-asad bhaiya :)

**Comment by anmode** [comment]
@0xabdulkhalid Prettier test is failing.. please fix it;
run `npx prettier --write .`

**Comment by 0xabdulkhalid** [comment]
Everything's are perfectly wrapped up now 🚀 

@anmode now it's time to merge this PR.

**Comment by anmode** [comment]
Cool, congrats.. Next PR would be level 3 for sure...

**Comment by 0xabdulkhalid** [comment]
Oops @anmode, you have forget to add `level` label for this merged PR.

Please make sure to add that necessary label otherwiser GSSOC won't take this PR in account.

Hope you understood.

**Comment by anmode** [comment]
sorry my bad! done

**Pull request merged**